js/ui/list/ui.list.edit.decorator.selection.js

```diff
@@ -95,9 +95,25 @@ registerDecorator(
             }
         },
 
-        moveFocusFromList: function(itemIndex) {
-            if(this._$selectAll && (itemIndex === 0 || itemIndex === this._list._getLastItemIndex())) {
-                this._selectAllCheckBox.focus();
+        handleKeyboardEvents: function(itemIndex, moveFocus) {
+            if(this._$selectAll && this._needMoveFocus(itemIndex, moveFocus)) {
+                this._list.option("focusedElement", undefined);
```

Comment:
List does not exist before first opening. Does tagBox crash if you press up/down before the tagBox is opened first time? (and when focusStateEnabled is false)